model/metricset.go

```diff
@@ -180,6 +185,18 @@ func (me *Metricset) Transform(ctx context.Context, _ *transform.Config) []beat.
 		fields["timeseries"] = common.MapStr{"instance": me.TimeseriesInstanceID}
 	}
 
+	// Metrics are stored in a "metrics" data stream. Metrics that include well-defined
+	// transaction/span fields (i.e. breakdown metrics, transaction and span metrics)
+	// will be stored separately from application and runtime metrics.
+	dataset := datastreams.NormalizeServiceName(me.Metadata.Service.Name)
```

Comment:
I thought the conclusion was that all apm documents would have an `apm` dataset prefix?

Reply:
Yes, I just hadn't updated the PR yet. I've updated it now.